src/Compilers/CSharp/Test/Semantic/IOperation/IOperationTests.cs

```diff
@@ -420,5 +420,248 @@ void M(A a)
             // Verify we return null operation for child nodes of member access expression.
             Assert.Null(model.GetOperation(expr.Name));
         }
+
+        [CompilerTrait(CompilerFeature.IOperation, CompilerFeature.Dataflow)]
+        [Fact]
+        public void MethodReference_NoControlFlow()
```

Comment:
>MethodReference_NoControlFlow [](start = 20, length = 29)

Consider creating a file dedicated to method references and moving the tests there.